spec/factories/reports.rb

```diff
@@ -39,6 +39,7 @@
 
     factory :standard_form_report, class: "Report::StandardFormReport" do
       form
+      group_by_tag true
```

Comment:
I'm not sure about making this the default as it's not the default in the UI. Why this change?